### PR TITLE
SEO edits for security center overview

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited title and description for length and keywords.

@DaniHalfin 

Edits from original PR 2155.

### CODE REVIEW

File: microsoft-365/security/mtp/overview-security-center.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview of the Microsoft 365 security center
-description: Describes the new home for monitoring and managing security across your Microsoft identities, data, devices, and apps.
+title: Overview - Microsoft 365 security center - Microsoft 365 security
```
Comment (levinec) [inline microsoft-365/security/mtp/overview-security-center.md]: Saying Microsoft 365 security twice seems pretty redundant, is that what we are doing @DaniHalfin?

Comment (DaniHalfin) [reply microsoft-365/security/mtp/overview-security-center.md]: @jdmartinez36 - you can remove the suffix here. There's already a suffix appended through build configuration ("Microsoft 365 security" is already configured as the suffix).

**Comment by jdmartinez36** [comment]
Hi @levinec and @DaniHalfin

I've removed the excess suffix from the title.

**Review by levinec — APPROVED** [review]

**Comment by tuco-ulf** [comment]
@DaniHalfin is this ready to merge?

**Pull request merged**